src/proxy/src/main/java/ee/ria/xroad/common/util/healthcheck/StoppableCombinationHealthCheckProvider.java

```diff
@@ -83,6 +84,9 @@ public HealthCheckResult get() {
                         .map(cacheResultFor(resultValidFor, errorValidFor, TimeUnit.SECONDS)),
                 checkServerConfDatabaseStatus()
                         .map(withTimeout(timeout, TimeUnit.SECONDS, "Server conf database status"))
+                        .map(cacheResultFor(resultValidFor, errorValidFor, TimeUnit.SECONDS)),
+                checkGlobalConfStatus()
```

Comment:
Global configuration check should be the first (cheap and fast unless the filesystem remote --> timeout and caching is necessary in edge cases).

Not changed in this issue, but the checkServerConfDatabaseStatus() does not work well -- due to caching in CachingServerConfImpl it does not actually check that database connection works once the security server identifier has been successfully loaded.

Reply:
Sounds reasonable. Moved it up in the list. 

Did not address the issue with `checkServerConfDatabaseStatus()` in this PR. Should we create a ticket about that?